### ISSUE TITLE
Latency consistency compared to LMAX Disruptor Java?

### ISSUE THREAD

**Issue by heri16** [Abc-Arbitrage/Disruptor-cpp#15]
Are there benchmark results on how consistent the latency is compared to LMAX Disruptor Java?

**Comment by mashoujiang** [comment]
same concern, could anyone answer?

**Comment by Andreone** [comment]
No, we haven’t compared the latencies. We don’t plan to work on this in the near future.
